packages/shared/src/components/search/SearchBar.tsx

```diff
@@ -0,0 +1,327 @@
+      <div className={classNames('relative', moreStyles.raiseLabelContainer)}>
+        <BaseField
+          {...props}
+          className={classNames(
+            'relative items-center px-3 h-16 rounded-14 border !border-theme-divider-tertiary !bg-theme-bg-primary',
+            className,
+            { focused },
+          )}
+          onClick={isMobile ? () => setReportModal(true) : focusInput}
+          data-testid="searchBar"
+          ref={ref}
+        >
+          {!isMobile && (
+            <AiIcon
+              size={IconSize.Large}
+              className="mr-3 text-theme-label-tertiary"
+            />
+          )}
+
+          {isMobile && (
+            <div className="flex-1 text-theme-label-tertiary">
+              { placeholder }
+            </div>
+          )}
+
+          {!isMobile && (
+            <FieldInput
+              disabled={disabled}
+              placeholder={placeholder}
+              name={name}
+              id={inputId}
+              ref={inputRef}
+              onFocus={(event) => {
+                onFocus();
+                externalOnFocus?.(event);
+              }}
+              onBlur={(event) => {
+                onBlur();
+                externalOnBlur?.(event);
+              }}
+              onInput={onInput}
+              autoFocus={autoFocus}
+              type="primary"
+              aria-describedby={describedBy}
+              autoComplete="off"
+              className={classNames(
+                'flex-1 caret-theme-status-cabbage',
+                getFieldFontColor({ readOnly, disabled, hasInput, focused }),
+              )}
+              required
+            />
+          )}
+
+          <div className="flex gap-3 items-center">
+            {hasInput && (
+              <Button
+                {...rightButtonProps}
+                className="btn-tertiary"
+                buttonSize={ButtonSize.Small}
+                title="Clear query"
+                onClick={onClearClick}
+                icon={<CloseIcon size={IconSize.Small} />}
+                disabled={!hasInput}
+              />
+            )}
+
+            {!isMobile && (
+              <div className="h-8 border border-theme-divider-quaternary" />
+            )}
+
+            <SimpleTooltip
+              content={
+                searchHistory.length === 0
+                  ? 'Your search history is empty'
+                  : 'See search history'
+              }
+            >
+              <div>
+                <Button
+                  {...rightButtonProps}
+                  className="btn-tertiary"
+                  buttonSize={ButtonSize.Small}
+                  title="Search history"
+                  onClick={seeSearchHistory}
+                  icon={<TimerIcon size={IconSize.Small} />}
+                  disabled={searchHistory.length === 0}
+                />
+              </div>
+            </SimpleTooltip>
+            {!isMobile && (
+              <Button
```

Comment:
This should be disabled with tooltip if no query is active.
<img width="177" alt="Screenshot 2023-08-02 at 11 59 31" src="https://github.com/dailydotdev/apps/assets/554874/d2fa8516-037e-4543-ba0d-d0fef3ea5a11">